How-to-develop-for-i18n.md

```diff
@@ -46,8 +46,14 @@ If a translation key is not used any more, you must delete it from `assets/i18n/
 
 To verify your changes to translation files locally, run the following command in a terminal:
 
+Python:
 ```
-    python -m scripts.run_backend_tests --test_target=core.controllers.base_test.I18nDictsTests
+python -m scripts.run_backend_tests --test_target=core.controllers.base_test.I18nDictsTests
+```
+
+Docker:
+```
+make run_backend_tests PYTHON_ARGS="--test_target=core.controllers.base_test.I18nDictsTests"
```

Comment:
throwing error

Reply:
@jaiakash Please fix this cmd

Reply:
i guess this command has some naming issues 
![Screenshot from 2024-02-10 13-27-06](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/829fff28-e505-41d1-a7e6-1d60fbb6757a)
as you can see run_backend_tests is not there instead that is run_tests.backend

Reply:
also the error is also stating that 
![Screenshot from 2024-02-10 13-31-53](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/ba632a50-c39e-4e54-8f1c-1837333357c7)

Reply:
@jaiakash @gp201  this is resolved